include/sacn/common.h

```diff
@@ -79,13 +88,28 @@ typedef struct SacnHeaderData
  * primary key for a network interface is simply a combination of the interface index and the IP
  * protocol used.
  */
-//TODO CHRISTIAN : This is identical to the RdmnetMcastNetintId.  They need to be merged into ETCPal as EtcPalMcastNetintId.
+// TODO CHRISTIAN : This is identical to the RdmnetMcastNetintId.  They need to be merged into ETCPal as
+// EtcPalMcastNetintId.
 typedef struct SacnMcastNetintId
 {
   etcpal_iptype_t ip_type; /*!< The IP protocol used on the network interface. */
   unsigned int index;      /*!< The OS index of the network interface. */
 } SacnMcastNetintId;
 
+/*! The functions on the source and receiver that involve creation and networking change
+ *   use this structure to indicate what interfaces were successfully created.
+ * */
+typedef struct SacnNetworkChangeResult
```

Comment:
I thought about that, but the gotcha for me was the "all" case, e.g. passing in NULL for the interface list.  No one has a clue as to what size that is going to be.

Before I went to the static array, in this structure the user had to give it the pointer to an array of interfaces to be filled in.  It seemed a little overboard to me, but I could move to that again and we can see how it looks.

Reply:
I think we should use the in/out array. I think in the "all"/NULL case, the user doesn't need feedback as to which NICs failed. People who want that feedback are probably passing explicit interface lists anyway.

Reply:
Yes, I like where this is going. I think we should use "netints" instead of "ifaces" to match naming elsewhere in sACN, RDMnet and EtcPal. Instead of SacnMcastInterfaceToUse, how about just SacnMcastInterface?

Reply:
Sam, did you have any other input on the netint change? If not, I'm considering this done.  And thanks for your help here -- this was one that didn't feel right but I couldn't see a way around it when trying to handle all cases.